docker/Dockerfile_aws

```diff
@@ -13,10 +13,7 @@ COPY requirements_webserver.txt ${FUNCTION_DIR}
 COPY serving/backend ${FUNCTION_DIR}
 COPY training/creating_dataset.py ${FUNCTION_DIR}/training/creating_dataset.py
 COPY saved_models ${FUNCTION_DIR}/saved_models
-COPY pathfinder_2e_data/pathfinder-bestiary.db  \
-    pathfinder_2e_data/pathfinder-bestiary-2.db  \
-    pathfinder_2e_data/pathfinder-bestiary-3.db  \
-    ${FUNCTION_DIR}/pathfinder_2e_data/
+COPY counterfactuals_datasets ${FUNCTION_DIR}/counterfactuals_datasets
```

Comment:
Hm, to raczej taka sugestia do przemyślenia, czy nie "ładniej" by się nazywał folder "counterfactual_datasets" - może wystarczy jeden rzeczownik w liczbie mnogiej, ale nie naciskam (zmiana dotyczyłaby oczywiście też folderu na repo)

Reply:
Czy potrzebujemy kopiować cały ten folder? Nie wystarczy jeden plik? Z tego co widzę server używa tylko basica